File: src/main/kotlin/com/linchpino/ai/repository/ResumeRepository.java
package com.linchpino.ai.repository;

import com.linchpino.ai.model.Resume;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ResumeRepository extends JpaRepository<Resume, Long> {

    int countAllByEmail(String email);
}

File: src/main/kotlin/com/linchpino/ai/repository/ResumeRepository.kt
package com.linchpino.ai.repository

import com.linchpino.ai.model.Resume
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ResumeRepository : JpaRepository<Resume?, Long?> {
    fun countAllByEmail(email: String?): Int
}
